include/modem/lte_lc.h

```diff
@@ -621,19 +621,19 @@ enum lte_lc_tau_triggered {
 /** CE level. */
 enum lte_lc_ce_level {
 	/** No repetitions or a small number of repetitions. */
-	LTE_LC_CE_LEVEL_0_NO_REPETITION		= 0,
+	LTE_LC_CE_LEVEL_0		= 0,
```

Comment:
You could align the numbers little closer to the name now that all the names are very short.

Reply:
The values are aligned with tabs and I though that one less tab would not be enough for `LTE_LC_CE_LEVEL_UNKNOWN`, but it seems that it is. I'll remove one tab from all.